templates/server/src/main/resources/archetype-resources/api/src/main/java/__packageInPathFormat__/general/common/api/to/UserProfileTo.java

```diff
@@ -34,4 +35,10 @@ public void setLogin(String login) {
     this.login = login;
   }
 
+  @Override
+  protected void toString(StringBuilder buffer) {
+
+	buffer.append(getClass().getSimpleName() + ": " + "login = " + login + ", hashcode = " + this.hashCode() + ";");
```

Comment:
please check your IDE. Seems you are not using Eclipse with proper formatter and save actions.

Reply:
In case you are still struggling with the IDE, then simply add two whitespaces to these 2 `toString` methods you have added.
BTW: Why did you add the hashCode to the toString result?